addon/services/redux.js

```diff
@@ -0,0 +1,50 @@
+import Ember from 'ember';
+import redux from 'redux';
+import reducers from '../reducers/index';
```

Comment:
@foxnewsnetwork ^^ any reason we can't/or shouldn't do this? on the surface it seems redundant as @brettburley mentioned